### ISSUE TITLE
opening the whole thing

### ISSUE THREAD

**Issue by aep** [softScheck/tplink-smartplug#36]
Thanks for decrypting the default password.
i soldered on a tty and got root on the thing.

in case you want me to try anything before i try flashing upstream linux, let me know.

**Comment by aep** [comment]
yup, i ported openwrt to it https://github.com/openwrt/openwrt/pull/694

**Comment by mirsella** [comment]
hey, what can you do with openwrt on a plug ?
